Dockerfile

```diff
@@ -6,6 +6,20 @@ ENV LANG=en_US.utf8 \
 
 EXPOSE 53 67 68 69 80 443 3000 3306 5910-5930 5432 8140 8443
 
+### Atomic/OpenShift Labels - https://github.com/projectatomic/ContainerApplicationGenericLabels
+LABEL name="acme/foreman-katello-systemd" \
```

Comment:
Should be resolved with https://github.com/RHsyseng/sturdy-chainsaw/pull/9/commits/d0ffb469dfa6a37ab73cb35bb42fa2c7de1870a3